crates/rpc/rpc/src/otterscan.rs

```diff
@@ -44,8 +45,16 @@ where
     }
 
     /// Handler for `ots_getInternalOperations`
-    async fn get_internal_operations(&self, _tx_hash: TxHash) -> RpcResult<Vec<InternalOperation>> {
-        Err(internal_rpc_err("unimplemented"))
+    async fn get_internal_operations(&self, tx_hash: TxHash) -> RpcResult<Vec<InternalOperation>> {
+        self.eth
+            .spawn_trace_transaction_in_block_with_inspector(
+                tx_hash,
+                TransferInspector::new(false),
+                |_tx_info, inspector, _, _| Ok(inspector.into_transfers()),
+            )
+            .await?
+            .map(|transfer_operations| transfer_operations.into_iter().map(Into::into).collect())
+            .ok_or_else(|| internal_rpc_err("failed to inspect transaction"))
```

Comment:
this is not required, can use `?` here I believe

Reply:
I can't use `?` on `Option<Vec<InternalOperation>` to get `Result<Vec<InternalOperation>, Err>`, so I use `unwrap_or_default` to the inside content and return with `Ok`